homeassistant/components/husqvarna_automower/number.py

```diff
@@ -111,7 +111,7 @@ async def async_setup_entry(
     """Set up number platform."""
     coordinator = entry.runtime_data
     entities: list[NumberEntity] = []
-
+    current_work_areas = {}
```

Comment:
We never update this one right?

Reply:
Yes, it's just for creating a set for each mower here:
https://github.com/home-assistant/core/blob/ad6fd5fa2928451e68e606adbbe496b5fa54226d/homeassistant/components/husqvarna_automower/number.py#L126

Reply:
But we should update if we got new values

Reply:
Ah okay, yes we do here:
https://github.com/home-assistant/core/blob/ad6fd5fa2928451e68e606adbbe496b5fa54226d/homeassistant/components/husqvarna_automower/number.py#L156